File: src/MaxConsecutiveOnes.java
public class MaxConsecutiveOnes {
    public int findMaxConsecutiveOnes(int[] array) {
        int maxOnes = 0, current = 0;
        for (int number : array) {
            if (number == 1) {
                current++;
                maxOnes = Math.max(maxOnes, current);
            } else {
                current = 0;
            }
        }

        return maxOnes;
    }

    public static void main(String[] args) {

    }
}

File: python/max_consecutive_ones.py
from typing import List


class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        max_ones, current = 0, 0
        for number in nums:
            if number == 1:
                current += 1
            else:
                max_ones = max(max_ones, current)
                current = 0
        max_ones = max(max_ones, current)
        return max_ones
